docroot/modules/custom/prisoner_hub_taxonomy_sorting/prisoner_hub_taxonomy_sorting.deploy.php

```diff
@@ -26,3 +27,14 @@ function prisoner_hub_taxonomy_sorting_deploy_set_field_default_values() {
     }
   }
 }
+
+
+/**
+ * Copy values from field_moj_date to field_release_date.
+ */
+function prisoner_hub_taxonomy_sorting_deploy_copy_moj_date_2() {
+  // @See https://drupal.stackexchange.com/a/250937/4831
+  Drupal::database()->query("INSERT INTO node__field_release_date SELECT * FROM node__field_moj_date;");
+  Drupal::database()->query("INSERT INTO node_revision__field_release_date SELECT * FROM node_revision__field_moj_date;");
```

Comment:
so this is some kind of migration - does this get run automatically?

Reply:
Yep this is a migration from one field to another (which is the only way we can rename the field).

And yes, it gets run automatically as part of the deployment.